lib/repl.js

```diff
@@ -1111,7 +1111,16 @@ function complete(line, callback) {
     }
 
     completionGroupsLoaded();
+  } else if (match = line.match(/.*\( *("|')(.*)/)) {
```

Comment:
should this also check for the `fs` prefix? i can imagine a lot of function calls take strings that aren't filenames

Reply:
Yes true, but what if someone had a code like:

```
const filesystem = require('fs')
filesystem.readFileSync(...
```

Then our `fs` regex will fail. I'm not sure, how to handle these cases.. Any ideas?

Reply:
its bound to `fs` by default so it might be safe to just assume it will be used that way. another option is to check if the prefix actually is an fs method by eager evaluating it (https://github.com/nodejs/repl/blob/df57668200fbe87469d9ef7947b46433f157b93a/src/index.js#L195-L211)